src/emulator.ts

```diff
@@ -151,26 +162,17 @@ export class EmacsEmulator implements IEmacsController, vscode.Disposable {
     );
 
     this.disposables = [];
+    this.commandRegistry = new EmacsCommandRegistry();
+    this.afterCommand = this.afterCommand.bind(this);
+
+    const killYanker = new KillYanker(this, killRing, minibuffer);
+    this.killYanker = killYanker;
+    this.registerDisposable(killYanker);
 
+    // Set up event listeners
```

Comment:
Such trivial commands are not needed.
Grouping semantically similar lines is enough.